applications/text_classification/hierarchical/analysis/evaluate.py

```diff
@@ -0,0 +1,254 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import functools
+import os
+import argparse
+
+import numpy as np
+from sklearn.metrics import accuracy_score, classification_report, f1_score
+
+import paddle
```

Comment:
```suggestion
from sklearn.metrics import accuracy_score, classification_report, f1_score
import paddle
```

paddle， skearn  这些都算第三方库，可不加空格